crest/Assets/Crest/Crest/Shaders/OceanInputs/Resources/ClipPainted.shader

```diff
@@ -0,0 +1,88 @@
+﻿// Crest Ocean System
```

Comment:
Just noting that these painted shaders be one shader? Blend Mode and Color Mask can be scripted. Or an includes file. Not sure if there is more sim specific code is needed in the future. Low priority anyhow.

Reply:
good shout. there are some other bits of difference, for example AnimWavesHeightPainted writes to the y channel of the data. some output vectors and some float. this feels solvable.. but needs more work. will leave this comment open.

Reply:
i don't think i'll get to this in the short term so i'm going to document in the future work notes in the PR description. it needs investigation whether we can get the required variation across shaders cleanly/elegantly, as noted in my previous comment.